### PR TITLE
Merge fenix quarantine

### PR DESCRIPTION
**Pull request by Pike**

### CODE REVIEW

**Comment by Pike** [comment]
lgtm, but figured I'd put this in D's bucket again.

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -472,15 +478,24 @@
     <!-- Notifications -->
     <!-- Text shown in snackbar when user deletes a collection -->
     <string name="snackbar_collection_deleted">Collection deleted</string>
+    <!-- Text shown in snackbar when user deletes a collection -->
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: Shouldn't the comment be "Text shown in snackbar when user *renames* a collection"? (vs "delete")

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -475,13 +489,37 @@
...
+    <string name="share_link_subheader">Share a link</string>
+    <!-- Sub-header in the dialog to share a link to another sync device -->
+    <string name="share_device_subheader">Send to device</string>
+    <!-- An option from the share dialog to sign into sync -->
+    <string name="sync_sign_in">Sign in to Sync</string>
+    <!-- An option from the share dialog to send link to all other sync devices -->
+    <string name="sync_send_to_all">Send to all devices</string>
+    <!-- An option from the share dialog to reconnect to sync -->
+    <string name="sync_reconnect">Reconnect to Sync</string>
+    <!-- An option from the share dialog to reconnect to sync -->
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: This is the same comment as above, and it seems like it should be updated.

Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml]: Update: Sawyer is updating this right now, so we should get that fix in with the next quarantine

**Pull request merged**